### PR TITLE
Objection: Update according to XCode's recommended settings.

### PR DESCRIPTION
**Pull request by byohay**

### CODE REVIEW

File: Objection.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1285,6 +1311,7 @@
 					"-lstdc++",
 					"-all_load",
 				);
+				PRODUCT_BUNDLE_IDENTIFIER = "atomicobject.${PRODUCT_NAME:rfc1034identifier}";
```
Comment (barakwei) [inline Objection.xcodeproj/project.pbxproj]: It's weird that this is in many places in this file instead of just in the project settings.

Comment (byohay) [reply Objection.xcodeproj/project.pbxproj]: 🤷‍♂ what do you want to do?

**Comment by byohay** [comment]
I looked into it, in other projects the `PRODUCT_BUNDLE_IDENTIFIER` is also determined in the target.

**Comment by byohay** [comment]
@barakwei moved `PRODUCT_BUNDLE_IDENTIFIER` to project level.

**Review by barakwei — APPROVED** [review]

**Pull request merged**